test/repo_tests/test_readme.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import os
+import re
+import pathlib
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
What is the purpose of this?

Reply:
Good call. It was leftover from a previous version. Removed.